ios/build.sh

```diff
@@ -154,29 +160,19 @@ if [ $DO_KEYMANAPP = true ]; then
       DEV_TEAM="DEVELOPMENT_TEAM=${DEVELOPMENT_TEAM}"
     fi
 
-    # To dynamically set the parameters in a way xcodebuild can use them, we need to construct the entire xcodebuild call as a string first.
-    BUILD_1="xcodebuild -quiet -project keyman/Keyman/Keyman.xcodeproj ${CODE_SIGN_IDENTITY} ${CODE_SIGNING_REQUIRED} ${DEV_TEAM} -target SWKeyboard -configuration ${CONFIG}"
-    BUILD_2="xcodebuild -quiet -project keyman/Keyman/Keyman.xcodeproj ${CODE_SIGN_IDENTITY} ${CODE_SIGNING_REQUIRED} ${DEV_TEAM} -target Keyman -configuration ${CONFIG}"
-
     if [ $DO_ARCHIVE = false ]; then
-      # Performs the actual build calls.
-      $BUILD_1
+      # To dynamically set the parameters in a way xcodebuild can use them, we need to construct the entire xcodebuild call as a string first.
```

Comment:
This is copied from the old code but I don't think we actually need to construct the command as a string.

Reply:
There's a reason it was a string in the old script.  It can be tricky - Xcode does very strange things when parsing parameters set by shell variables, and I was never able to get it right at the time.

I might put in a little time on it, but dropping the string is low priority.